sdk/storage/azure-storage-files-datalake/test/ut/datalake_file_client_test.cpp

```diff
@@ -588,14 +591,10 @@ namespace Azure { namespace Storage { namespace Test {
     EXPECT_TRUE(downloadResult.Value.Details.EncryptionContext.HasValue());
     EXPECT_EQ(encryptionContext, downloadResult.Value.Details.EncryptionContext.Value());
     // Assert ListPaths
-    auto paths = m_fileSystemClient->ListPaths(false).Paths;
-    auto iter = std::find_if(
-        paths.begin(), paths.end(), [&fileName](const Files::DataLake::Models::PathItem& path) {
-          return path.Name == fileName;
-        });
-    EXPECT_NE(paths.end(), iter);
-    EXPECT_TRUE(iter->EncryptionContext.HasValue());
-    EXPECT_EQ(encryptionContext, iter->EncryptionContext.Value());
+    auto paths = directoryClient.ListPaths(false).Paths;
```

Comment:
Now you checked encryption context of listed item returned by `DataLakeDirectoryClient::ListPaths`. How about those returned by `DataLakeFileSystemClient::ListPaths`?